### DISCUSSION TITLE
I can't get free XCH from the faucet

### DISCUSSION THREAD

**Discussion by Spytitan** [Chia-Network/chia-blockchain#8102] [Support]
I have tried to get free XCH several times for join the pool. But the coin still hasn't been in the wallet

**Comment by BrandtH22** [comment]
@Spytitan , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

Note - it can take some time for the wallet to sync. If the icon(s) in the top right corner are not green and indicate synced then we will want to ensure the client is syncing properly, if these are green and indicate synced then we will want to verify that the funds were sent to the correct xch address. Let us know which method of troubleshooting is needed either by reopening this ticket or joining us in our discord.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
